powerlevel9k.zsh-theme

```diff
@@ -337,6 +337,55 @@ prompt_background_jobs() {
   fi
 }
 
+# Segment to indicate hdd available level.
+set_default POWERLEVEL9K_HDD_USAGE_ONLY_WARNING false
+set_default POWERLEVEL9K_HDD_USAGE_WARNING_LEVEL 10
+set_default POWERLEVEL9K_HDD_USAGE_CRITICAL_LEVEL 5
+prompt_hdd_usage() {
```

Comment:
You named the segment `hdd_usage`, but it actually shows the amount of free space, right? I think another name would make that clearer.
Otherwise it could be nice to add an option, so that the user can decide what to show free space or usage..